src/components/ItemList/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import { updateItemDate } from '../Utils/firestore';
+import { differenceInMilliseconds } from 'date-fns';
+
+function ItemList({ itemName, docId, lastPurchasedDate }) {
+  const [isChecked, setIsChecked] = useState(false);
+  const [isDisabled, setIsDisabled] = useState(false);
+  const [token] = useState(() => window.localStorage.getItem('tcl18-token'));
+
+  const MarkProductPurshased = (id) => {
+    updateItemDate(token, id)
+      .then(() => {
+        console.log('product updated');
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  };
+
+  useEffect(() => {
+    if (lastPurchasedDate && isDisabled === false) {
+      const hoursAtTheMoment = differenceInMilliseconds(
+        new Date(),
+        lastPurchasedDate.toDate(),
+      );
+      console.log(hoursAtTheMoment)
```

Comment:
watch out for sneaky left over console logs!

Reply:
That part of code was refactored.